INSTALL.md

```diff
@@ -2,6 +2,7 @@
 
   * [Pre-requisites](#pre-requisites)
   * [Simple Installation](#simple-installation)
+  * [Installation as Python Package from PyPI](#installation-as-python-package-from-pypi)
```

Comment:
I don't think this should come before the "Installation via Package Manager" section.  The historical problems with pip, and pipx's close relationship to pip, are enough to make me not trust pipx much until it proves itself it.  As such, it should not be high on the recommendation list to start.

Reply:
I agree. The order should reflect Unix/Mac (not Windows). I will wait a bit if you have further comments below and then change.

Reply:
I don't understand the Unix/Mac comment.  Some package managers in the package managers list are Windows-specific.  And pipx works on Unix/Mac.  So the order of those two things says nothing about operating system.  Further, the rationale for putting pipx later was because of its relationship to pip, which specifically made things worse for Windows users (not Mac or Linux users), when we suggested it once upon a time, so putting it later is very specifically in consideration of Windows.